crates/astria-sequencer-relayer/src/finalization_pipeline/mod.rs

```diff
@@ -0,0 +1,194 @@
+use std::{
+    collections::HashMap,
+    mem,
+};
+
+use astria_sequencer_types::SequencerBlockData;
+use tendermint::Hash;
+
+pub(crate) mod head_candidate;
+pub(crate) mod pipeline_item;
+
+pub(crate) use head_candidate::HeadCandidate;
+pub(crate) use pipeline_item::PipelineItem;
+
+// head of the sequencer chain as observed on cometbft.
+#[derive(Default)]
```

Comment:
remove the `Default`, there isn't a natural default for this

Reply:
I believe it is in fact `Fork` since requirements for this code was that blocks coming from sequencer always point to [forks of] the canonical chain and always arrive in sequential order relative to height